.github/workflows/cross-ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         rust: [stable]
-        feature: [default]
+        feature: [default, self-update]
```

Comment:
This is quite annoying, as this will create more runs overall. The alternative is, that I change the `cross-ci` action, to determine, that if it checks `rustic`, to add this feature.